designs/administrator/4.6/guided-tours/index.md

```diff
@@ -0,0 +1,102 @@
+
+## Accessing tours
+Tours can be accessed on both the Administrator and Developer perspectives by going to the **Help menu** and selecting **Guided Tours**. Each of the perspectives will also surface links to the tours in other places around the console. For more detail around how these links will be surfaced in the Developer perspective, view the Developer side documentation.
+
+In the Administrator perspective, a badge will appear on the Overview page informing users that guided tours are available. When clicked, the badge will redirect users to the Guided Tours page.
+
+![1](img/admin1.png)
+
+The Guided Tours page will use a catalog-like design with each tour being shown on a tile. Tiles will provide a brief description of the tour, a time estimate for completing the tour, as well as prerequisites for completing the tour, if any.
+
+![2](img/admin2.png)
+
+If a user clicks on a tour tile, the tour side panel will open. The introductory panel will give users more information and allows the user to start the tour using the **Start tour** button.
+
+![3](img/tour1-screen0.png)
+
+## Tour permissions
+For 4.6, tour permissions will be as follows:
+* If a user can install Operators, show the first Serverless tour (Explore Serverless) which walks users through installing the Serverless Operator.
+* If a user cannot install Operators, do not show the first Serverless tour. Instead show an empty state. The empty state can inform users that tours will be added in future.
+* If an Operator has been installed, show the second Serverless tour (Serverless Applications) which walks users through deploying a Serverless application.
+* Any tour that has been completed should remain on the Guided Tours page with the option to **Review the tour** or **Restart the tour**.
+
+## Tour 1: Explore Serverless
+The first tour walks users through installing the Serverless Operator by completing a series of 3 tasks. Each task will have a number of sub steps that the user should execute in order to successfully complete the tour. Any links within the tour instructions will redirect users to that page in the console.
+
+### Check your work
+There will not be any validation from the console that users have successfully completed any sub steps or tasks. Instead, there will be an optional **Check your work** feature that asks users to validate that the tasks have been completed.
+
+![4](img/check1.png)
+
+The **Check your work** feature will appear after a user clicks **Next**. The user may click **Next** again to skip the validation step, or can read the instructions to make sure they are following the tour correctly.
+
+![5](img/check2.png)
+
+If the user selects **Yes**, the Check your work alert will turn green.
+
+![6](img/check3.png)
+
+Users can click Next again to advance to the next screen, where they will see that the first task has been marked as completed and the second task is shown.
+
+![7](img/tour1-screen6.png)
```

Comment:
I think it would remain in its neutral state if the user doesn't select anything. One idea that we liked is that users could use the Check your work feature almost as a bookmark, to indicate to themselves when they come back which steps they've done. So if a user is just clicking through everything to see a preview of the tour before completing it, we probably don't want to automatically select success.

Reply:
We thought about not allowing users to proceed if they check "No" (I believe that's how Solution Explorer does it), but we also wanted the feature to be optional and thus not be a blocker. Although you have a great point, I think choosing "No" _is_ the user opting in to the Check your work feature... 😄Maybe if the user selects "No", we disable the Next button until they select "Yes". Curious others' thoughts here too!

Reply:
This question came up during one of our feedback sessions - is 'next' tied to a required action?

This may be due to my lack of knowledge of patternfly, but the selection looks like a radio button. In that case, defaulting to 'yes', and enabling the 'next' button makes sense. If the user switches the value to 'no' disabling 'next' could make sense.

With that said, the mockups below allow a user to go through the guided tour just to see what the next steps are and go back to repeat the necessary steps as desired. There is something nice about the feature being optional and allowing users to move back and forth through the steps organically.

Reply:
Would a neutral state for a previous step have blue text, the same numbered blue circle, and no sentences beneath it? I see a green example here and red example below, but not a neutral example for a previous step. Sorry, kind of a nit, maybe that'll be in the developer side's design.

Letting users proceed without interacting with the Check your work feature makes sense to me 👍. Defaulting to "Yes" shouldn't be needed, the two radio buttons can both be unselected to start, as shown.

But yeah, if a user opts into the feature's hand-holding and selects "**No**, the Serverless operator's status isn't `Succeeded`, it's actually in a scary-looking red error state" I wonder if part of this feature's hand-holding should include _not_ letting those newbie users proceed to the next step's instructions (which might not be applicable since the previous step wasn't done correctly). 🤷‍♂️

Reply:
It looks like it's possible for a user to say "No" even when the answer is "Yes" and then the next step says it's not installed but it actually is. Has there been any thoughts around how this could be confusing? Can we not detect if the installation was a success and not require them to confirm it? If it fails maybe we tell them why.